src/Resources/views/_scripts.html.twig

```diff
@@ -1,6 +1,6 @@
 <script type="text/javascript">
     var monsieurbizSearchPlugin = {
-        instantEnabled: true,
+        instantEnabled: {{ setting('monsieurbiz.search', 'instant_search_enabled__monsieurbiz_product') ? 'true' : 'false' }},
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** This is OK if you only index products. If I also have taxons or other entities in ES, we need to check that instant search is disabled for all entities.
Perhaps you should create a twig function to loop through all documentable entities and check the configuration. ‏

Reply:
We need to build a form dynamically with all documentable entities with a default value to `true` if the value is not set yet

Reply:
@maximehuran a form here? But this is not a form, we don't generate any input fields. I am confused why you suggest a form here.